integration/test/performance_tracker/transmitter/transmitter.go

```diff
@@ -0,0 +1,237 @@
+package transmitter
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbiface"
+)
+const(
+	TIME_CONSTANT = 300.0 // this const is in seconds , 5 mins
+)
+type TransmitterAPI struct{
+	DynamoDbClient dynamodbiface.DynamoDBAPI
+	DataBaseName      string // this is the name of the table when test is run
+}
+// this is the packet that will be sent converted to DynamoItem
+type Metric struct{
+	Average float64
+	StandardDev float64
+	Period	int 	//in seconds
+	Data []float64
```

Comment:
Do we want to store the raw data points as part of the record? What value does it provide?